winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    if (e.ColumnName != "ID")
+        return;
+
+    e.Range.CellStyle.Font.Size = 12;
+    e.Range.CellStyle.Font.Color = ExcelKnownColors.Pink;
+    e.Range.CellStyle.Font.FontName = "Segoe UI";
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-column-cell-customization.png" alt="Customizing Cell Values based on Column name while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the `ID` column cells are customized while exporting.
+
+## Customize exported workbook and worksheet
+
+The TreeGrid can be exported to Excel using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). You can refer to the [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating the workbooks and sheets after exporting.
+
+### Set borders
+
+You can set the borders to Excel cells by directly accessing worksheet after exporting the data.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].UsedRange.BorderInside(ExcelLineStyle.Dash_dot, ExcelKnownColors.Black);
+workBook.Worksheets[0].UsedRange.BorderAround(ExcelLineStyle.Dash_dot, ExcelKnownColors.Black);
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-cell-border-style.png" alt="Changing Border Style in Exported Excel for WinUI TreeGrid" width="100%" Height="Auto"/>
+
+#### Enabling Filters
+
+Show filters in the exported worksheet by enabling a filter for the exported range in the worksheet.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+workBook.Worksheets[0].AutoFilters.FilterRange = workBook.Worksheets[0].UsedRange;
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-filter.png" alt="Filters on Exported Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+
+While using the `stacked headers`, the filter option enables for the stacked header cell. To avoid this, specify the `range` based on the Stacked headers count to show the filter icon in the header cell.

```

Comment:
While using the `stacked headers,` the filter option enables the stacked header cell. To avoid this, specify the `range` based on the Stacked headers count to show the filter icon in the header cell.

Reply:
Updated